src/components/typography/typography.component.tsx

```diff
@@ -46,214 +53,89 @@ export interface TypographyProps extends SpaceProps, ColorProps {
   display?: string;
   /** Override the list-style-type */
   listStyleType?: string;
-  /** Override the white-space type */
+  /** Override the white-space */
   whiteSpace?: string;
-  /** Override the word-wrap type */
+  /** Override the word-wrap */
   wordWrap?: string;
-  /** Override the text-overflow type */
+  /** Override the text-overflow */
   textOverflow?: string;
   /** Apply truncation */
   truncate?: boolean;
+  /** Override the color style */
+  color?: string;
+  /** Override the backgroundColor style */
+  backgroundColor?: string;
+  /** Override the bg value shorthand for backgroundColor */
+  bg?: string;
+  /** Override the opacity value */
+  opacity?: string | number;
+  /** Override the withMargin boolean */
+  withMargin?: boolean;
+  /** Override the className value */
+  className?: string;
```

Comment:
```suggestion
/** @private @ignore */
className?: string;
```
I think while some components do support `className`, this is usually a private prop as we don't encourage consumers to use classes for custom css styling.

Reply:
Makes sense